SPEC.md

```diff
@@ -319,17 +319,92 @@ The maximum CPU speed is 100, setting any number higher is the same as setting
 to 100. The game operates at 10 frames per second meaning a fully overclocked
 CPU will execute 1000 instructions per second, 100 per frame.
 
+### Battery Check
+
+A robot can know how much battery it has:
+
+```
+push8 #IO_BATTERY
+io
+popf batteryLevel
+```
+
+The float on the stack will be between `0.0` and `1.0` describing how much

```

Comment:
Since battery is currently not capped (and according to what @krisives  told me, BATTERY_MAX is more meant as BATTERY_START), this is not true. You can have more than 100% battery.